src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/DatabaseErrorPageOptions.cs

```diff
@@ -12,6 +14,28 @@ namespace Microsoft.AspNetCore.Builder
     public class DatabaseErrorPageOptions
     {
         /// <summary>
+        /// Constructs a new DatabaseErrorPageOptions.
+        /// </summary>
+        public DatabaseErrorPageOptions() { }
+
+        /// <summary>
+        /// Constructs and configures a new DatabaseErrorPageOptions.
+        /// </summary>
+        /// <param name="configureOptions">The configuration actions that will be applied.</param>
+        public DatabaseErrorPageOptions(IEnumerable<IConfigureOptions<DatabaseErrorPageOptions>> configureOptions) : this()
```

Comment:
Hold off on making any changes for a bit if you can we are still in the final iterations of the new pattern. I will update all the repos in the next few days so we should soon be in a new consistent state for options. 

But the basic general idea is there should be a default constructor that any defaults. And one "options" ctor that takes the IConfigureOptions and applies them.